File: Python/linear_search.py
print ("Enter number of elements")
n = int(input())
print ("Enter array")
l=list(map(int,input().split()))
print ("Enter element to be searched")
k = int(input())

flag = 0
for i in range(n):
	if l[i] == k:
		flag = 1
		pos = i
		break 

if flag == 0:
	print("Element not present")
else:
	print("Element is present at index", str(pos))

File: C++/linear_search.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
  int n,num;
  printf("Enter the total no. of elements: \n");
  scanf("%d",&n);
  int a[n];
  printf("Enter the elements: \n");
  for(int i=0;i<n;i++){
    scanf("%d",&a[i]);
  }
  printf("Enter the number you want to search: \n");
  scanf("%d",&num);
  for(int i=0;i<n;i++){
    if(a[i] == num){
      printf("Element found at location: %d",i+1);
      break;
    }else if(i == n-1){
      printf("Element not found.");
    }
  }
  return 0;
}
